app.R

```diff
@@ -268,37 +277,37 @@ server <- function(input, output, session) {
       leaflet.extras::addSearchGoogle(searchOptions(autoCollapse = FALSE, minLength = 8)) %>%
       # This is to add assembly boundaries and to be able to popup the information
       leaflet.extras::addGeoJSONChoropleth(json_data,
-        valueProperty = "AREASQMI",
-        scale = c("white", "red"),
-        mode = "q",
-        steps = 4,
-        padding = c(0.2, 0),
-        labelProperty = "name",
-        popupProperty = propstoHTMLTable(
-          props = c("name", "description", "altitudeMode", "extrude"),
-          table.attrs = list(class = "table table-striped table-bordered"),
-          drop.na = TRUE
-        ),
-        color = "#43a858", weight = 1, fillOpacity = 0.7,
-        highlightOptions = highlightOptions(
-          weight = 2, color = "#9c4e57",
-          fillOpacity = 1, opacity = 1,
-          bringToFront = TRUE, sendToBack = TRUE
-        ),
-        pathOptions = pathOptions(
-          showMeasurements = TRUE,
-          measurementOptions =
-            measurePathOptions(imperial = TRUE)
-        ),
-        group = "District boundaries"
+                                           valueProperty = "AREASQMI",
+                                           scale = c("white", "red"),
+                                           mode = "q",
+                                           steps = 4,
+                                           padding = c(0.2, 0),
+                                           labelProperty = "name",
+                                           popupProperty = propstoHTMLTable(
+                                             props = c("name", "description", "altitudeMode", "extrude"),
+                                             table.attrs = list(class = "table table-striped table-bordered"),
+                                             drop.na = TRUE
+                                           ),
+                                           color = "#43a858", weight = 1, fillOpacity = 0.7,
+                                           highlightOptions = highlightOptions(
+                                             weight = 2, color = "#9c4e57",
+                                             fillOpacity = 1, opacity = 1,
+                                             bringToFront = TRUE, sendToBack = TRUE
+                                           ),
+                                           pathOptions = pathOptions(
+                                             showMeasurements = TRUE,
+                                             measurementOptions =
+                                               measurePathOptions(imperial = TRUE)
+                                           ),
```

Comment:
seems formatting changes. Can you set the formatted in local in a way that does not reformat again

Reply:
done